installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{- define "service.image" -}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
```

Comment:
This could be slightly restructured to have the full docs comment before the helper function.
```suggestion
{{/*
Return the proper serivce image name
{{- include "common.images.image" ( dict "imageRoot" .Values.yourService.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}
*/}}
{{- define "service.image" -}}
```